voice/bxml/verbs/record.md

```diff
@@ -0,0 +1,48 @@
+{% method %}
+## XML: `<Record>`
+The Record verb allows a segment of audio to be recorded during a call. At the end of the recording, a [Record Complete](../callbacks/recordComplete.md) event is generated.
+
+### Attributes
+
+| Attribute                    | Description                                                                                                                                                                                                            |
+|:-----------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| recordCompleteUrl            | (optional) URL to send the [Record Complete](../callbacks/recordComplete.md) event to and request new BXML.                                                                                                            |
+| recordCompleteMethod         | (optional) The HTTP method to use for the request to `recordCompleteUrl`. GET or POST. Default value is POST.                                                                                                          |
+| recordingAvailableUrl        | (optional) URL to send the [Recording Available](../callbacks/recordingAvailable.md) event to.                                                                                                                         |
+| recordingAvailableMethod     | (optional) The HTTP method to use for the request to `recordingAvailableUrl`. GET or POST. Default value is POST.                                                                                                      |
+| username                     | (optional) The username to send in the HTTP request to `recordCompleteUrl` or `recordingAvailableUrl`. If username and password are specified, then https is required.                                                 |
```

Comment:
```suggestion
| username                     | (optional) The username to send in the HTTP request to `recordCompleteUrl` or `recordingAvailableUrl`. If specified, the `recordingAvailableUrl` must be TLS-encrypted (i.e., `https`).                                                 |
```

Reply:
Specifying just one also requires `https`, right?

Reply:
If only one is specified then it is ignored completely :-o
See `HttpUtils#generateBasicAuthentication`.